code/datums/diseases/advance/advance.dm

```diff
@@ -424,4 +424,9 @@ var/list/advance_cures = 	list(
 		src << "<a href='?_src_=vars;Vars=\ref[D]'>[D.name] - [D.holder]</a>"
 */
 
+/datum/symptom/heal/proc/Heal(mob/living/M, datum/disease/advance/A)
+	var/get_damage = sqrt(20-"stage_rate")*(2+rand())

```

Comment:
I would also recommend reading about BYOND's associative lists. Which, in other languages, are called a HashMap or Dictionary.

Difference between these two kinds of lists is that, with the normal list, you can access the element by the order it is in the list:

```C++
var/list/my_list = list()
my_list += 5
my_list[1] // Would return 5, since it is the 1st in the list
```

And with the associative list, which is what the ```properties``` variable is, you can add your values differently and retrieve them in any order. You just need to give it a key so it knows what value to give you. Like so:

```C++
var/list/my_list = list()
my_list["my_key"] = 5
my_list["my_key"] // Will give you back your 5.
```

There is a shorter way to write this, like so:

```C++
var/list/my_list = list("my_key" = 5)
```

Which is what I do in the advance.dm file.